content/posts/November-2020.md

```diff
@@ -23,16 +23,36 @@ draft: false
 
 ## Week 47 (November 16th - November 20th)
 
-- [Matej] fixed [an
+- ~~[Matej] fixed [an
   issue](https://github.com/packit/packit-service/issues/774) with Packit CLI,
   enabling recursive search for spec-files
-  ([packit#1005](https://github.com/packit/packit/pull/1005)).
+  ([packit#1005](https://github.com/packit/packit/pull/1005)).~~ It turned
+  out, this change made it very easy to [deplete GitHub API quota], so it was
+  reverted.
 - [Laura] made Copr builds to be part of the default jobs. This will cause
   build jobs to explicitly run, and display their own result flags in PRs,
   next to the test results. We hope this to be a more friendly behaviour for
   our new users ([packit#1024](https://github.com/packit/packit/pull/1024)).
 
+## Week 48 (November 23rd - November 27th)
+
+- [Laura] [fixed a bug], where Packit Service failed to trigger Copr builds
+  before triggering the tests in case triggering Copr builds was configured
+  for a different event.
+- Thanks to [Matej], Packit Service will now [comment on commits] in case jobs
+  failed after a merge. This way failures for jobs triggered by branch events
+  (merge or push) should become more noticeable.
```

Comment:
@mfocko Is this correct?

Reply:
Not really, it comments when there are unsupported targets, failed update of copr project or no builder permissions

Reply:
tbh it's a nice idea though, shouldn't be that hard to add :+1: